killercoda-sample/karmada-CLI-installtion-example/foreground.sh

```diff
@@ -0,0 +1,63 @@
+echo waiting for init-background-script to finish
+
+function installKind() {
+  touch installKind.sh
+  echo "wget https://github.com/kubernetes-sigs/kind/releases/download/v0.17.0/kind-linux-amd64" >> installKind.sh
```

Comment:
It would be preferable to declare the version as a variable in the header of this file.